tutorials/java/chapters/ch07.adoc

```diff
@@ -0,0 +1,221 @@
+:imagesdir: images
+
+## Build and Deploy Java EE 7 Application
+
+https://github.com/javaee-samples/javaee7-simple-sample[Java EE 7 Simple Sample] is a trivial Java EE 7 sample application.
+
+[[Build_Application]]
+### Build Application
+
+. Clone the repo:
+
+  git clone https://github.com/javaee-samples/javaee7-simple-sample.git
+
+. Build the application:
+
+  mvn clean package
+
+### Start Application Server
+
+Start WildFly server as:
+
+[source, text]
+----
+docker run --name wildfly -d -p 8080:8080 -v /Users/youruser/javaee7-simple-sample/target:/opt/jboss/wildfly/standalone/deployments/:rw jboss/wildfly
```

Comment:
Might be a good opportunity to use `docker volumes` (named volumes) here, rather than bind mounting onto the host's filesystem.